Fix ember-remodal doesn't remove padding-right when the modal closes in some cases

al3xnag opened the pull request:
Fixes #263.

danielpost committed:
Add aria-hidden to wrapper, trap Tab key focus inside modal

danielpost committed:
Actually add aria-hidden

al3xnag committed:
Fix ember-remodal doesn't remove padding-right when the modal closes in some cases

Fixes #263.

danielpost commented:
I think this repo is dead, I have a PR from Feb 27th that hasn't been answered at all.

al3xnag committed:
Fix ember-remodal doesn't remove padding-right when the modal closes

In case when body has `padding-right: ... !important;` somewhere in stylesheets.

al3xnag committed:
Merge remote-tracking branch 'danielpost/master'

al3xnag committed:
Update dist folder

al3xnag committed:
Revert Tab key focus trapping: it does not work properly

Revert commit #90a2ffaf35e33c2193c586bc0a318c9d4e4f715e
